proxy/src/schema.rs

```diff
@@ -1,4 +1,6 @@
 use juniper::{FieldResult, ParseScalarResult, ParseScalarValue, RootNode, Value};
+use radicle_registry_client::{Error as RegistryError};
+use substrate_subxt::error::{Error as SubstrateError};
```

Comment:
`radicle_registry_client::Error` is a type alias for `substrate_subxt::Error`. So for the compiler these two types are the same. This means that you only need to import the one. We’re trying to move away from the subxt errors with https://github.com/radicle-dev/radicle-registry/issues/109